tasks/cockpit-tasks-webhook.yaml

```diff
@@ -5,20 +5,20 @@ items:
 - kind: ReplicationController
   apiVersion: v1
   metadata:
-    name: release
+    name: tasks-webhook
   spec:
     replicas: 1
     selector:
-      infra: cockpit-release
+      infra: cockpit-tasks
```

Comment:
And this would be particularly confusing, as we have a separate RC for the actual task execution. So let's just name the yaml file, and all labels like "webhook"?